handbook/engineering/README.md

```diff
@@ -204,6 +204,8 @@ Fleet's documentation for contributors can be found in the [Fleet GitHub repo](h
 ## Release process
 
 - [Release freeze period](#release-freeze-period)
+- [Freeze day](#freeze-day)
+- [Merging during freeze](#merging-features-during-freeze)
```

Comment:
This will be a broken link, because the heading doesn't match.  I recommend removing the little table of contents lists.  They have great intentions, but we spend far too much time cleaning them up.  It increases the numbers of lines changed in diffs.  It's not worth the maintenance to have these.

Reply:
```suggestion

```